examples/jsm/exporters/EXRExporter.js

```diff
@@ -189,22 +189,30 @@ function buildInfoDT( texture, options = {} ) {
 
 }
 
-function getPixelData( renderer, rtt, info ) {
+async function getPixelData( renderer, rtt, info ) {
 
 	let dataBuffer;
 
-	if ( info.type === FloatType ) {
+	if ( renderer.isWebGLRenderer ) {
 
-		dataBuffer = new Float32Array( info.width * info.height * info.numInputChannels );
+		if ( info.type === FloatType ) {
+
+			dataBuffer = new Float32Array( info.width * info.height * info.numInputChannels );
+
+		} else {
+
+			dataBuffer = new Uint16Array( info.width * info.height * info.numInputChannels );
+
+		}
+
+		renderer.readRenderTargetPixels( rtt, 0, 0, info.width, info.height, dataBuffer );
```

Comment:
@gkjohnson My plan was to use `WebGLRenderer.readRenderTargetPixelsAsync()` here as well but using it produces a WebGL feedback loop warning in the browser console when exporting the PMREM in [misc_exporter_exr](https://threejs.org/examples/misc_exporter_exr). The screen also gets black.

> misc_exporter_exr.html:1 [.WebGL-0x128005ba800] GL_INVALID_OPERATION: Feedback loop formed between Framebuffer and active Texture.

I did not find the root cause so far so the code still uses the sync approach for now. It would be great if you could provide a second pair of eyes and have a look^^.

Reply:
Do you have a minimal repro for how to cause the issue? Does it just require setting or rendering to RenderTarget A and then calling "readRenderTargetPixels"?

Reply:
Just switch to `readRenderTargetPixelsAsync()` in `EXRExporter` any try to export the PMREM in `misc_exporter_exr`. That should trigger the feedback loop.